lib/BackbeatConsumer.js

```diff
@@ -457,6 +457,7 @@ class BackbeatConsumer extends EventEmitter {
             entry: { topic, partition, offset,
                      key: key && key.toString(), timestamp },
             committableOffset,
+            offsetLedger: this._offsetLedger.toString(),
```

Comment:
may also need logging on 'marked consumed entry'

Reply:
You got it!